src/content/1.7/basics/keeping_up-to-date/migration.md

```diff
@@ -0,0 +1,678 @@
+---
+title: Migration
+weight: 20
+---
+
+# How to: Migrate
```

Comment:
I think we can change this to "How to migrate from a previous version"

Reply:
What about "How to migrate to the last major version"?